### ISSUE TITLE
Product titles replaced with slug during live edit

### ISSUE THREAD

**Issue by Ezyweb-uk** [microweber/microweber#696]
Whilst updating product pages in live edit mode in version 1.1.20, the product title is being replaced with the url slug (capitals replaced with lowercase, spaces replaced with hyphens). Using the New World template, a small change to a product description field in live edit mode changes the product title (without clicking the green save button). 

On a change to the product description and click away (eg to another browser tab) the auto save appears to be replacing the product title with the url slug,  although the description change is not saved until the green save button is clicked.

In file _src\Microweber\Providers\Content\ContentManagerHelpers.php_ in _function save_from_live_edit_ if _$post_data['id']_ is not set then it looks like line 696 replaces the title with the url slug. A log of $post_data gives the following output in which 'id' is indeed missing:

```
    Array
    (
        [field_data_0] => Array
            (
                [attributes] => Array
                    (
                    [class] => edit typography-area
                    [field] => content_body
                    [rel] => content
                    [itemprop] => dateModified
                    [content] => 2020 Dec 08
                )

            [html] =>                                                         <p class="element" id="element_1607435020136">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. </p>
                                                        <p class="element" id="element_1607435020135">Product description goes here ...<br></p>
                                                    
        )

)
```
I'm not sure where the ajax post fetches the id from, but I can see the html body classes include _page-id-8 post-id-21 content-id-21_ and the userfiles\templates\template-name\layouts\shop_inner.php wraps the product data in a div with id _shop-content-21_ and the p-tags include ids.

I tested editing a regular page rather than a product and logged the post data, but no id key for that either. Looking in the js code it looks like maybe the keydown event triggers `mw.drag.save()` which calls `mw.drag.collectData(edits) ` followed by `mw.drag.coreSave` in _userfiles\modules\microweber\api\liveedit\drag.js_ ; function coreSave then makes the ajax post to the save_edit api function, but no reference to id.

**Comment by peter-mw** [comment]
fixed in 1.2.0

**Comment by Ezyweb-uk** [comment]
Thanks Peter!
